### ISSUE TITLE
Change icon left click click behaviour

### ISSUE THREAD

**Issue by autoteelar** [vector-im/element-web#23606]
#### What would you like to do?
make left click toggle hide/show app
#### Why would you like to do it?
because it makes more sense and every other app does it
#### How would you like to achieve it?

make right click stay the same, but instead of making left click a copy of what right click does, just make it quickly show/hide

you can also very accidentially close the app accidentially just by clicking once, as linux has some weird feature wher eyour mouse clicks stay until release. 

tldr is just extremely annoying like that and i think it should be this way. i see absolutely 0 downsides and only upsides to changing this.

**Comment by autoteelar** [comment]
simply just double clicking the icon as of now quits it, as the quit button spawns right under the mouse, although this is a WM thing, its still impractical and worth mentioning again another reason why this change is important to end users

**Comment by t3chguy** [comment]
> make left click toggle hide/show app

This is how it is programmed to work, but the various compatibility layers in Electron/Chromium make it not behave as expected sometimes. 

https://github.com/vector-im/element-desktop/blob/develop/src/tray.ts#L61

This is an upstream issue https://github.com/electron/electron/issues/14941

**Comment by autoteelar** [comment]
thank you t3chguy, so its an electron thing and its already coded in element nice, hopefully we see these things worked out :)

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/element-web/issues/11772

**Comment by t3chguy** [comment]
@autoteelar unfortunately not, Linux has multiple tray icon implementations, fixing one will regress another, Element also relies on Electron handling it given handling all the world's tray icon implementations ourselves would expend a lot of developer time.
